client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -135,6 +135,13 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			this.props.updateFields( { [ name ]: ! this.props.fields[ name ] } );
 		};
 
+		handleAutosavingToggle = name => () => {
```

Comment:
Ok, I'll merge this as it is now, and we can afterwads, do any change that's appropriate